internal/handlers/commonerrors/write.go

```diff
@@ -21,13 +21,28 @@ import (
 	"github.com/FerretDB/FerretDB/internal/util/must"
 )
 
+// writeError represents protocol write error.
+// It required to build the correct write error result.
+// The index field is optional and won't be used if it's nil.
```

Comment:
What does it mean by nil index field? It's int32 :thinking:

Reply:
That was an old comment; updated